package.json

```diff
@@ -1,35 +1,43 @@
 {
-  "name": "tibber-httpclient",
-  "version": "1.13.11",
-  "description": "",
-  "main": "./lib/index.js",
-  "types": "./lib/index.d.ts",
-  "scripts": {
-    "build": "yarn build-ts && yarn tslint",
-    "build-ts": "tsc -d",
-    "tslint": "tslint -c tslint.json -p tsconfig.json"
+ "name": "tibber-httpclient-got",
+ "version": "2.0.0",
+ "main": "./dist/src/index.js",
+ "author": "Tibber",
+ "license": "MIT",
+ "scripts": {
+  "lint": "eslint src/ --ext .js,.ts,.json",
```

Comment:
ESLint isn't installed, so perhaps remove?
Actually, you are going to have to change the CircleCI configuration too, right? Since it probably runs `yarn lint`.

Reply:
I forgot to add it, only copied in the config files. Maybe VSCode picked those up anyhow and I didn't notice. I'm adding ESLint.

Reply:
Oh wow, when copying and pasting to this repo, I didn't see the hidden config files. Must have been sleeping!?!